module/Decision/view/email/authorization_received.phtml

```diff
@@ -1,7 +1,7 @@
 <?php $lang = 'en'; ?>
-For English, see below.
+For English, please see below.<br /><br />
 
-L.s.<br /><br />
+L.S.,<br /><br />
```

Comment:
Shouldn't the line break be just `<br>` ?

Reply:
> Shouldn't the line break be just `<br>` ?

That would have my preference, but due to the existing use of `<br />` I opted for that one. Possibly it was once decided that `<br />` looked better or was more compatible with some mail clients.
I think consistency is even more important than preferences, so if we were to change it, it should be changed everywhere.

Reply:
Yeah, I guess we can do it in some other PR at some point.